RocketChat/Rocket.Chat.ReactNative
# fix: Remove react-native-keycommands

## Proposed changes

iPad shortcuts has been broken for quite some time and we need to come up with a fix for that.
We tried a few solutions, but they didn't work.
For now, we're dropping support to it and we'll resume the work on this in the future.

## Issue(s)	

Depends on https://github.com/RocketChat/react-native/pull/6

## How to test or reproduce

- Shortcuts shouldn't crash iPad
- Run `yarn` and check if `react-native` is fetching the correct version inside node_modules https://github.com/RocketChat/react-native/pull/6/files#diff-0c169cfc1eff9cf64ba1b661e7e7788c7a9ae6c1caaafafa6b6c2f41fbef0468L21

## Screenshots

## Types of changes




## Checklist



## Further comments

[NATIVE-93]

[NATIVE-93]: https://rocketchat.atlassian.net/browse/NATIVE-93?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
